solution-base/mongodb/charts/mongodb-sharded/values.yaml

```diff
@@ -1327,12 +1351,12 @@ shardsvr:
     ##
     containerSecurityContext:
       enabled: true
-      seLinuxOptions: null
+      seLinuxOptions: {}
       runAsUser: 1001
-      runAsGroup: 0
+      runAsGroup: 1001
       runAsNonRoot: true
       privileged: false
-      readOnlyRootFilesystem: false
+      readOnlyRootFilesystem: true
```

Comment:
could this affect some procedure, e.g. when exec in the pod to run mongosh and generate a file (collection dump or similar)?